client/runtime.go

```diff
@@ -221,6 +225,7 @@ func (r *Runtime) Submit(operation *runtime.ClientOperation) (interface{}, error
 	if err != nil {
 		return nil, err
 	}
+	request.timeout = r.timeout
```

Comment:
I think this effectively overwrites the timeout set on a per request basis.
I think instead it should get passed into the `newRequest` method and replace this value: https://github.com/go-openapi/runtime/blob/94927f8c9742791a9aa46c6c3965f0a37025f41d/client/request.go#L43